src/components/common/carousel/carousel-desktop/carousel-desktop.tsx

```
@@ -24,11 +24,12 @@ export default function CarouselDesktop({ slides, options }: PropType) {
         <div className='embla__container'>
-          {slides.map((slide, index) => (
-            <div className='embla__slide' key={index}>
-              <div className='embla__slide__number'>{slide}</div>
-            </div>
-          ))}
+          {slides &&
+            slides.map((slide, index) => (
+              <div className='embla__slide' key={index}>
+                <div className='embla__slide__number'>{slide}</div>
+              </div>
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)